doc/users/dflt_style_changes.rst

```diff
@@ -513,6 +555,8 @@ Legends
 
 - The legend now has rounded corners by default.
 
+- And a lighter border, I think? cf. #6770. (@QuLogic)
```

Comment:
-> "The legend defaults now include rounded corners, a lighter boundary, and partially transparent boundary and background."